cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -277,13 +277,13 @@ public CloseableIterator<StructuredRow> multiScan(Collection<Range> keyRanges,
    * Generates a SELECT query for scanning over all the provided ranges. For each of the range, it generates a where
    * clause using the {@link #appendRange(StringBuilder, Range)} method. The where clause of each range are OR together.
    * E.g.
-   *
+   * <p>
    * SELECT * FROM table WHERE key1 in (?,?) AND key2 in (?,?)
    * OR ((key3 >= ?) AND (key3 <= ?)) OR ((key4 >= ?) AND (key4 <= ?)) LIMIT limit
    *
    * @param keyFields a map from field name to field values that the query has to match with
-   * @param ranges the list of ranges to scan
-   * @param limit number of result
+   * @param ranges    the list of ranges to scan
```

Comment:
unnecessary whitespace change.

Reply:
I imported the [coding standard ](https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard)and it re-formatted this file. I guess it increased a bit readability? Since it highlights the param and de-highlight the description:
![image](https://user-images.githubusercontent.com/20428112/188759705-fe1263f5-0eb6-4aa8-8b3e-d36f68165102.png)